coffee_cmd/src/cmd.rs

```diff
@@ -70,6 +70,9 @@ pub enum CoffeeCommand {
         #[arg(short, long, action = clap::ArgAction::SetTrue)]
         verify: bool,
     },
+    /// tipping a plugins developer.
+    #[clap(arg_required_else_help = false)]
+    Tip { plugin: String, amount_msat: u64 },
```

Comment:
we allow the user to tip just one plugin at a time but we display the output in a table
I think we should only use tables when we are trying to organize similar data